tests/docs/contrib/e2e-environment.md

```
@@ -73,7 +73,12 @@ Additionally, you can directly pass the env variables.
 | KAI_QE_S3_BUCKET_NAME  | (Optional) AWS S3 bucket name (only needed if running evaluation)                                                                                                                                                                                          |
 | PARASOL_API_KEY        | (Optional) API key for MaaS provider (Only used if working with https://maas.apps.prod.rhoai.rh-aiservices-bu.com/)                                                                                                                                        |
 | UPDATE_LLM_CACHE       | (Optional) When set to true, the offline tests will run in online mode and generate new cached data for the configured provider. Will require provider credentials to run.                                                                                 |
-| ANALYZER_BINARY_PATH   | (Optional) Absolute path used by [custom-binary-analysis.test.ts](../../e2e/tests/base/custom-binary-analysis.test.ts) to define a custom path for the analyzer binary                                                                                     |
+| ANALYZER_BINARY_PATH   | (Optional) Absolute path used by [custom-binary-analysis.test.ts](../../e2e/tests/base/custom-binary-analysis.test.ts) to define a custom path for the analyzer binary  
+
+| TEST_REPO_URL   | (Optional) Default is 'https://github.com/konveyor-ecosystem/coolstore'
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)